### DISCUSSION TITLE
"Author URL is not accessible" error when "Enable Authors" is unchecked

### DISCUSSION THREAD

**Discussion by SpaceToastKADK** [Automattic/wordpress-activitypub#479] [Q&A] [answered]
I unchecked Enable Authors under Settings, as I'd prefer to serve up the whole blog in one feed, but doing so triggers the dreaded Site Health screen "Author URL is not accessible" warning. Author pages are excluded from caching in WP Super Cache. Disabling plugins besides ActivityPub doesn't help. Checking Enable Authors again makes the error go away.

Any idea what might be going on? I haven't been able to find a way around this.

**Comment by pfefferle** [comment, marked as answer]
This is simply an error in the plugin. I have to disable the author check if the authors are disabled. Sorry about that!
